includes/db.inc

```diff
@@ -35,9 +35,14 @@ function islandora_usage_stats_pid_id($pid) {
  */
 function islandora_usage_stats_pid_to_db($pid) {
```

Comment:
@adam-vessey Regarding this line:
`$object = $pid instanceof AbstractObject ? $pid : islandora_object_load($pid);`

This seems a bit confusing, as `$pid instanceof AbstractObject` should always return false since the `$pid` getting passed in is a string, not an object. On returning false, it would assign the `$pid` string value to `$object`, which is misleading AND affects the way the values are inserted into the SQL call that comes afterward (`->fields(array('pid' => $pid, 'label' => $object->label))` if `$object` is a real object, and `->fields(array('pid' => $pid, 'label' => $object))` if `$object` is just a PID string).